.chloggen/ct_update-default-grace-period-to-30s.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: target allocator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: set default target allocator grace period to 30 seconds
+
+# One or more tracking issues related to the change
+issues: [3989]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  The target allocator now waits for 30 seconds before reallocating targets from a collector which isn't Ready.
```

Comment:
Please call out exact config option that has changed `collectorNotReadyGracePeriod`